storage/app/releases.dist.php

```diff
@@ -113,6 +113,16 @@
                 72, // WASM-4
             ],
         ],
+        Emulators::PCSX2 => [
```

Comment:
This constant still isn't defined. Nor do we want to define one. Just use `101`.